src/components/layout/yearInReview/teamBlock.tsx

```diff
@@ -17,7 +17,7 @@ const TeamBlock: React.FC<TeamBlockProps> = ({
   children,
 }) => {
   return (
-    <div className='flex flex-col md:flex-row gap-10 mb-10'>
+    <div className='flex flex-col md:flex-row gap-5 md:gap-10 mb-10'>
```

Comment:
does this effect other yir pages?

Reply:
Yes, just the 2022 `moreEffectiveTeams` component which now has a margin to the edge of the screen on mobile that matches the other 2022 content around it